pymars/soln2cti.py

```diff
@@ -501,6 +514,33 @@ def build_species_string():
                 f.write(falloff_str)
             except IndexError:
                 f.write('\n           )\n\n')
+        elif equation_type == 'PlogReaction':
+            reaction_string = Template('#  Reaction $m\n' + \
+                            'pdep_arrhenius( \"$equation_string\",\n')
+            reaction_string = reaction_string.substitute(
+                    m=m,
+                    equation_string=equation_string)
+            sum_coeffs = sum(equation_object.reactants.values())
+            for rate_line in equation_object.rates:
+                pressure = str('{:f}'.format(rate_line[0]/ct.one_atm))
+                arrhenius = build_arrhenius((sum_coeffs,rate_line), equation_type)
+                arrhenius = arrhenius[1:-1]
+                reaction_string = Template(
+                        reaction_string + 
+                        "               [($pressure, 'atm'), $Arr],\n")
+                reaction_string = reaction_string.substitute(
+                        pressure=pressure,
+                        Arr = arrhenius)
+            if equation_object.duplicate is True:
+                reaction_string = reaction_string + \
+                                '               options=\'duplicate\')\n\n'
+            else:
+                reaction_string = reaction_string[:-1]
+                reaction_string = reaction_string + ')\n\n'
+            f.write(reaction_string)
+        else:
+            print('Error: Unknown reaction type')
+            exit()
```

Comment:
This should raise an error or warning, rather than brute-force close. Maybe `raise ValueError('Unknown reaction type')`